File: java/06_vulnerable_components.java
// A06 - Vulnerable Components. VULN: outdated dependency versions in pom snippet.
/*
 * VULN: A06 - known CVEs in these versions
 * <dependency>
 *   <groupId>org.apache.logging.log4j</groupId>
 *   <artifactId>log4j-core</artifactId>
 *   <version>2.14.0</version>
 * </dependency>
 * <dependency>
 *   <groupId>com.fasterxml.jackson.core</groupId>
 *   <artifactId>jackson-databind</artifactId>
 *   <version>2.9.8</version>
 * </dependency>
 */
public class VulnerableComponents {}

File: go/06_vulnerable_components.go
// A06 - Vulnerable Components. VULN: go.mod pins with known CVEs.
/*
 * VULN: A06
 * require (
 *   github.com/gin-gonic/gin v1.3.0
 *   golang.org/x/crypto v0.0.0-20181108071528-e3df2e7401b0
 * )
 */
package main

func VulnerableComponents() {}
